Blog/view/logIn.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+session_start();
+
+//include('Blog/include/functions.php');
+include('Blog/include/loginFunctions.php');
```

Comment:
as I referenced above, I'd just make a single `include_all.php` function which you include at the top of every main web page. That file should include all of your other include files. That way you always have access to all of your functions without having to think about it.